src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -4,15 +4,67 @@
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
 
+import javafx.scene.control.Tooltip;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
 
 public class FieldNameLabel extends Label {
+    private Tooltip tip;
+    private static final String AUTHOR = "author";
```

Comment:
There is no need for these Strings. Please use `org.jabref.model.entry.field.StandardField#AUTHOR` etc.

Reply:
Yes, I got it. Thank you for giving the advice!